### ISSUE TITLE
Improve color selection

### ISSUE THREAD

**Issue by Wilker-a** [markusfisch/WaveLinesWallpaper#3]
just a small suggestion, can you add more ways to select the colors, like RGB, CMY, etc., as well as a text field for placing exact tones? also, idk if it would be better to show the values in the range of 0-255 instead of .0-1.0, but i would like to see it. 

thank you \^-\^

it's a nice wallpaper btw :3

**Comment by markusfisch** [comment]
Thanks for contributing! 😄

Yes, of course, I understand having multiple ways to specify a color would be better. I just kept things simple to start with but I really should add a full color picker.

**Comment by markusfisch** [comment]
Version 1.7.0 allows entering a color in hex notation now.

**Comment by Wilker-a** [comment]
thank you \^-^

**Comment by markusfisch** [comment]
Yes, the color selection can still be improved and I keep it open to remind me of that 😄
